src/plugins/hetero/executable_network.cpp

```diff
@@ -790,7 +773,15 @@ InferenceEngine::Parameter HeteroExecutableNetwork::GetConfig(const std::string&
 }
 
 InferenceEngine::Parameter HeteroExecutableNetwork::GetMetric(const std::string& name) const {
-    if (EXEC_NETWORK_METRIC_KEY(SUPPORTED_METRICS) == name) {
+    if (ov::supported_properties == name) {
+        return decltype(ov::supported_properties)::value_type{
+            ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RO},
+            ov::PropertyName{ov::model_name.name(), ov::PropertyMutability::RO},
+            ov::PropertyName{ov::optimal_number_of_infer_requests.name(), ov::PropertyMutability::RO},
+            ov::PropertyName{ov::execution_devices.name(), ov::PropertyMutability::RO},
+            ov::PropertyName{ov::device::properties.name(), ov::PropertyMutability::RW},
+            ov::PropertyName{ov::device::priorities.name(), ov::PropertyMutability::RW}};
```

Comment:
I suppose after compilation `ov::device::properties ` and especially `ov::device::priorities ` cannot be changed?

Reply:
Yes. Fixed